drivers/timer/sam0_rtc_timer.c

```diff
@@ -102,10 +110,21 @@ static inline void rtc_sync(void)
 static uint32_t rtc_count(void)
 {
 #ifdef RTC_READREQ_RREQ
-	RTC0->READREQ.reg = RTC_READREQ_RREQ;
-#endif
+	u32_t ret;
+
+	rtc_sync();
+	if ((RTC0->READREQ.reg & RTC_READREQ_RCONT) == 0) {
```

Comment:
Use the READREQ.bit.RCONT version instead.

Reply:
done